priv/repo/migrations/20170801141406_change_system_variable_value_to_text_adds_required.exs

```diff
@@ -0,0 +1,10 @@
+defmodule CoursePlanner.Repo.Migrations.ChangeSystemVariableValueToTextAddsRequired do
+  use Ecto.Migration
+
+  def change do
+    alter table(:system_variables) do
+      modify :value, :text
+      add :required, :boolean
```

Comment:
it would be nice to make this field not null, it is not very nice to have 3-state booleans, and have a default value as well, I think it can be true by default